axelor-human-resource/src/main/java/com/axelor/apps/hr/service/timesheet/TimesheetServiceImpl.java

```diff
@@ -498,27 +499,38 @@ public Timesheet getCurrentTimesheet() {
   }
 
   @Override
-  public Timesheet getCurrentOrCreateTimesheet() {
+  public Timesheet getCurrentOrCreateTimesheet() throws AxelorException {
     Timesheet timesheet = getCurrentTimesheet();
     if (timesheet == null) {
       timesheet =
           createTimesheet(
-              AuthUtils.getUser(), appHumanResourceService.getTodayDateTime().toLocalDate(), null);
+              AuthUtils.getUser().getEmployee(),
```

Comment:
getEmployee can return null here, so the timesheet creation will fail.

Reply:
Already check for employee == null inside createTimesheet method. So, it will throw the error.